extra/minishift/README.md

```diff
@@ -1,7 +1,5 @@
 
-**Under constructions**  This lab is being re-worked to run on Minishift
-
-![Overview](images/arch_diagram_new.png)
+![Overview](../../images/diagram_general.png)
 
 # MicroProfile Lab with Open Liberty and Minishift
```

Comment:
The "Before you begin" section needs to be updated. I suggest you copy this section from the OKD lab as it has info about Operators and ....

Reply:
Also "Deploying a MicroProfile application in a Minishift cluster" needs to be updated to not mention about Open Liberty helm chart. Please ensure ICP is not mentioned across the lab.

Reply:
Hi Navid! I made the changes and ensure that ICP is not mentioned across the lab.